### ISSUE TITLE
Consider adding IPS Wrapper specs to 2.2 Data Provenance and Security

### ISSUE THREAD

**Issue by awatson1978** [HL7/standard-personal-health-record-ig#18]
See the following spec that could have a lot of useful, granular, detail on encryption standards for the SPHR spec:
https://github.com/HealthIntersections/ips-wapper-draft/blob/main/spec.md

**Comment by awatson1978** [comment]
Connectathon notes:

IPS Wrapper
Russian Nesting Doll Model
What happens if they get deeply nested.
> Something has probably gone deeply wrong; complex medical condition, clinicians arguing, etc.  
> Can we collapse the annotations of deeply nested into a single annotation level?
kludgy, but gets the job done.

Example:  rejected goal, due to contraindication
Transaction bundles could queue up server updates to any future system that runs across the Addendum.

Recommended:  DocumentReference if PDF attached; Composition resource otherwise.  
Annotation authoring utility would need to be aware of request methods:  POST/PUT/PATCH/DELETE

Transaction format could be very useful.

```
{
    "fullUrl" : "urn:uuid:61ebe359-bfdc-4613-8bf2-c5e300945f0a",
    "resource" : {
      "resourceType" : "Patient",
      "text" : {
        "status" : "generated",
        "div" : "<div xmlns=\"http://www.w3.org/1999/xhtml\">Some narrative</div>"
      },
      "active" : true,
      "name" : [{
        "use" : "official",
        "family" : "Chalmers",
        "given" : ["Peter",
        "James"]
      }],
      "gender" : "male",
      "birthDate" : "1974-12-25"
    },
    "request" : {
      "method" : "POST",
      "url" : "Patient"
    }
  }
```

Conclusion:  any system that handles Annotations/Corrections may benefit from parsing transaction format.  
Basically what the Annotation is doing is a _Virtual Patch_
